### PR TITLE
added in _check_for_lto_index_dir function

### PR DESCRIPTION
**Pull request by BleakleyMcD**
necessary to work with writelto

### CODE REVIEW

File: mmfunctions [hunk 1]
```diff
@@ -28,6 +28,15 @@ _check_for_lto_md5_flags(){
     fi
 }
 
+_check_for_lto_index_dir(){
+    if [ -z "${LTO_INDEX_DIR}" ] ; then
+        echo "LTO_INDEX_DIR is not set in mmconfig. Default of ${HOME}/Documents/lto_indexes will be used."
+        echo "If a different location is desired please set this in mmconfig"
+        LTO_MD5_FLAGS="${HOME}/Documents/lto_indexes"
```
Comment (retokromer) [inline mmfunctions]: Should it not be `LTO_INDEX_DIR` instead of `LTO_MD5_FLAGS`?

File: mmfunctions [hunk 2]
```diff
@@ -28,6 +28,15 @@ _check_for_lto_md5_flags(){
...
+    else
+        LTO_INDEX_DIR="${LTO_INDEX_DIR}"
+    fi
+}
```
Comment (retokromer) [inline mmfunctions]: Please, add an empty line between the two functions.

File: mmfunctions (hunk 1 above)
Comment (BleakleyMcD) [inline mmfunctions]: yes. oops!

File: mmfunctions (hunk 2 above)
Comment (BleakleyMcD) [inline mmfunctions]: ok.

**Comment by BleakleyMcD** [comment]
@dericed :)

File: mmfunctions [hunk 3]
```diff
@@ -28,6 +28,16 @@ _check_for_lto_md5_flags(){
     fi
 }
 
+_check_for_lto_index_dir(){
+    if [ -z "${LTO_INDEX_DIR}" ] ; then
+        echo "LTO_INDEX_DIR is not set in mmconfig. Default of ${HOME}/Documents/lto_indexes will be used."
+        echo "If a different location is desired please set this in mmconfig"
+        LTO_INDEX_DIR="${HOME}/Documents/lto_indexes"
+    else
```
Comment (dericed) [inline mmfunctions]: 36-37 don't do anything, can be removed

**Comment by retokromer** [comment]
+1
Sorry, @dericed, I missed that :-(

**Comment by BleakleyMcD** [comment]
fixed this in new pr #178  instead of here. oops!

**Pull request merged**